src/components/auth/SignUpScreen.js

```diff
@@ -1,290 +1,303 @@
-                      placeholderTextColor={COLORS.lightblue}
-                      returnKeyType='go'
-                      autoCapitalize='none'
-                      autoCorrect={false}
-                      secureTextEntry={this.state.hidePassword2}
-                      ref='FourthInput'
-                      onChangeText={value => this.onChangeText('password_confirmation', value)}
+              <Text style={styles.header}>Olá!</Text>
+              <Text style={styles.paragraph}>
+                Excepteur commodo deserunt eu ad labore labore qui ullamco
+                cillum consectetur incididunt eiusmod Lorem. Sunt do ipsum id
+                officia officia.
+              </Text>
+              <Form style={styles.formContainer}>
+                <Item style={styles.itemStyle}>
+                  <Input
+                    placeholder="Name"
+                    placeholderTextColor={COLORS.forestgreen}
+                    returnKeyType="next"
+                    autoCapitalize="none"
+                    autoCorrect={false}
+                    onSubmitEditing={(event) => {
+                      this.refs.SecondInput._root.focus();
+                    }}
+                    onChangeText={(value) => this.onChangeText("name", value)}
+                  />
+                </Item>
+                <Item style={styles.itemStyle}>
+                  <Input
+                    placeholder={i18n.t("Email")}
+                    placeholderTextColor={COLORS.forestgreen}
+                    returnKeyType="next"
+                    autoCapitalize="none"
+                    autoCorrect={false}
+                    keyboardType={"email-address"}
+                    ref="SecondInput"
+                    onSubmitEditing={(event) => {
+                      this.refs.ThirdInput._root.focus();
+                    }}
+                    onChangeText={(value) => this.onChangeText("email", value)}
+                  />
+                </Item>
+                <Item style={styles.itemStyle}>
+                  <Input
+                    placeholder={i18n.t("Password")}
+                    placeholderTextColor={COLORS.forestgreen}
+                    returnKeyType="next"
+                    autoCapitalize="none"
+                    autoCorrect={false}
+                    secureTextEntry={this.state.hidePassword}
+                    ref="ThirdInput"
+                    onSubmitEditing={(event) => {
+                      this.refs.FourthInput._root.focus();
+                    }}
+                    onChangeText={(value) =>
+                      this.onChangeText("password", value)
+                    }
+                  />
+                  <Ionicons
+                    style={
+                      this.state.hidePassword == false
+                        ? styles.showIcon
+                        : styles.hideIcon
+                    }
+                    name="ios-eye"
```

Comment:
Please use **assets/icons/ic_show_text** and **assets/icons/ic_hide_text** instead of ionicons

Reply:
I think the icons may have to be SVG. I'll double check this.

Reply:
they should scale well enough

Reply:
If they don't, we may want to scrap them => the other day I actually drew out an SVG icon similar to it but ran into a little roadblock when trying to get it into the project. I tabled it to get to review stage faster.

Reply:
React Native doesn't support SVG without an external package from what I can tell. Typically, people just do pngs using icon, icon@2x, icon@3x and scale the best fitting sized image.

I think the only benefit with using SVG in react native would be file size.

But yes, if it looks fuzzy when you scale it, we can switch back to using the vector icons package.

Reply:
I am pushing something up right now, tbh I cannot tell whether it is fuzzy or not: 

<img width="392" alt="Screen Shot 2020-07-08 at 5 17 36 PM" src="https://user-images.githubusercontent.com/22000425/86982787-f3c30980-c13e-11ea-9743-a9b3ec58d31e.png">

Reply:
I think that looks perfect